src/code42cli/cmds/securitydata.py

```diff
@@ -214,9 +214,11 @@
 def risk_indicator_callback(filter_cls):
     def callback(ctx, param, arg):
         if arg:
-            mapped_arg = (risk_indicator_map[arg[0]],)
+            mapped_args = ()
+            for i in range(len(arg)):
+                mapped_args += (risk_indicator_map[arg[i]],)
```

Comment:
This could be simplified to:

```
mapped_args = tuple(risk_indicator_map[i] for i in arg)
```

Reply:
Nice! I knew there was a better way than a for loop but I couldn't quite sort it out